source/query/filter.txt

```diff
@@ -359,11 +374,44 @@ The query returns the following document:
       "_id": { "$oid": "5e349915cebae490877d561d" },
       "name": "Andrea Le",
       "email": "andrea_le@fake-mail.com",
+      "school": {
+         "name": "Northwestern"
+      },
+      "version": 5,
+      "scores": [ 85, 95, 75 ],
+      "dateCreated": { "$date": "2003-03-26" }
+   }
+
+Match by Embedded Field
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following query filter finds the
+document with the ``school.name`` subfield of "Northwestern":
+
+.. code-block:: shell
+
+   { "school.name": "Northwestern" }
+
+
+The query returns the following document:
+
+.. code-block:: JSON
+   :copyable: false
+   :emphasize-lines: 4
```

Comment:
I think we want the emphasis on the school field? Currently it's on the email field.